lib/hanami/events/adapter/redis.rb

```diff
@@ -15,11 +15,12 @@ def initialize(params)
           @logger = params[:logger]
           @subscribers = []
           @thread_spawned = false
+          @stream = params[:stream] || STREAM_NAME
         end
 
         def broadcast(event_name, payload)
           @redis.with do |conn|
-            conn.lpush(STREAM_NAME, {
+            conn.lpush(@stream, {
```

Comment:
I've rewritten `initialize` to this:
`params.fetch(:stream, DEFAULT_STREAM)` which feels good enough for this case. 
I like to see the default value at initialization point.